CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Repository Maintainers
+* @launchdarkly/team-sdk-js
```

Comment:
This repo should meet the requirements for this code owners. 

I wonder if something isn't quite right in terraform?

Reply:
Yeah, looks like there is a piece missing.

Reply:
This repo has all the correct tags, but it doesn't appear in the query. So I will move forward with this while we investigate in parallel.